### ISSUE TITLE
How do I use tflint?

### ISSUE THREAD

**Issue by chimeworld** [terraform-linters/tflint#468]
I'm trying to use `tflint` on the CLI and in Vim and VS Code.

If I just run `tflint` in folder containing `main.tf`, nothing happens.

In Vim using Ale, I get the following:
```
Error detected while processing function <SNR>67_NeoVimCallback[29]..<lambda>3[1]..
<SNR>65_ExitCallback[28]..<SNR>64_HandleExit[39]..ale_linters#terraform#tflint#Hand
le:
line    3:
E714: List required
```
In VS Code, with the Terraform extension, it just says `Linting failed, more information in the output tab.` with nothing appearing in the output tab.

**Comment by wata727** [comment]
Unfortunately, I'm not sure about TFLint integration for Ale and VS Code... Please open an issue in the repositories.

Probably, changing JSON output in v0.11 may have an impact. I can only promise that the results in the CLI are correct.

**Comment by ali-alsabbah** [comment]
@wata727 the problem is that running tflint in the CLI outputs nothing

**Comment by wata727** [comment]
Ah, understood. First of all, the default CLI behavior does not output anything, which is the intended behavior. It means "No issues found".

Next, probably third-party tools such as VS Code and Ale are running with `--format json`. At this time, the output is valid JSON as follows:

```json
{
  "issues": [],
  "errors": []
}
```

I guess that these tools discard the TFLint output because of an internal error. This is the responsibility of the tool side.

**Comment by ali-alsabbah** [comment]
Well, I had a file that showed issues with `terraform validate` and the output was still blank.

I'll try again and see if I still have the issue.

**Comment by ali-alsabbah** [comment]
Here's an example.

```
➜  managed-policies git:(8202) ✗ terraform validate

Error: "policy" contains an invalid JSON: invalid character '"' after array element

  on main.tf line 21178, in resource "aws_iam_policy" "PIIHive":
21178: resource "aws_iam_policy" "PIIHive" {


➜  managed-policies git:(8202) ✗ tflint                 
➜  managed-policies git:(8202) ✗ 
```

**Comment by wata727** [comment]
TFLint doesn't detect all problems that terraform validate can detect. So this behavior is naturally possible.

**Comment by ali-alsabbah** [comment]
So what is an example of something that tflint will definitely pick up?

**Comment by wata727** [comment]
For example, it will report about invalid instance type. See README.

**Comment by wata727** [comment]
Close this due to no activity.

**Comment by jowet** [comment]
has the same issue.
Initializing terraform via `terraform init` solved it for me.
